ci-operator/config/openshift/aws-ebs-csi-driver-operator/openshift-aws-ebs-csi-driver-operator-master.yaml

```diff
@@ -26,6 +26,7 @@ images:
   from: src
   to: aws-ebs-csi-driver-operator-test
 promotion:
+  disabled: true
```

Comment:
Since the images are not promoted, I think all tests should be removed from this file, because they will test image built from csi-operator repo.